### PR TITLE
fix: should disable db connection ssl when some url params are provided

### PR DESCRIPTION
**Pull request by adrien2p**
**What**
Enable ssl identification from the client url query params as well

### CODE REVIEW

File: packages/core/utils/src/modules-sdk/load-module-database-config.ts [hunk 1]
```diff
@@ -31,7 +31,9 @@ function getDefaultDriverOptions(clientUrl: string) {
   }
 
   if (clientUrl) {
-    return clientUrl.match(/localhost/i) ? localOptions : remoteOptions
+    return clientUrl.match(/localhost|ssl_mode=(disable|false)/i)
```
Comment (carlos-r-l-rodrigues) [inline packages/core/utils/src/modules-sdk/load-module-database-config.ts]: We could check `127.0.0.1` as well. And maybe also if ssl_mode is true?

**Review by carlos-r-l-rodrigues — APPROVED** [review]

File: packages/core/utils/src/modules-sdk/load-module-database-config.ts (hunk 1 above)
Comment (adrien2p) [reply packages/core/utils/src/modules-sdk/load-module-database-config.ts]: let me improve

Comment (adrien2p) [reply packages/core/utils/src/modules-sdk/load-module-database-config.ts]: done 👍

**Review by sradevski — APPROVED** [review]

**Review by riqwan — APPROVED** [review]
nice!

**Pull request merged**